packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -109,20 +133,29 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
                 newNamePath = newNamePath.replace(/\\/g, "/"); // Added to cover Windows backslash issue
```

Comment:
```
let newNamePath = path.join(parentPath + newName);
newNamePath = newNamePath.replace(/\\/g, "/"); // Added to cover Windows backslash issue
```

Can the code to get `newNamePath` be replaced with :
`const newNamePath = path.join(parentPath, newName);`

@katelynienaber was there any special reason we used the `+` in the `path.join` argument and then did the manual replace afterwards?

Reply:
Addressed by pulling in commit 4a052cb after scrum discussion.